### PR TITLE
feat(#129): StanfordNLP in UnitTestIsNotVerb lint

### LINKED ISSUE #129: Migrate from OpenNLP to the latest version of similar lib with POS tags
Check this one: https://github.com/objectionary/lints/pull/126#issuecomment-2531215735

### PR DESCRIPTION
**Pull request by h1alexbel**
In this pull request I've replaced `opennlp` with `stanford-corenlp` library, and made `UnitTestIsNotVerb` lint stricter by allowing only `VBZ` (Verb, 3rd person singular present) verbs in the test names.

closes #129

### CODE REVIEW

**Comment by h1alexbel** [comment]
@volodya-lombrozo please check this one

File: src/main/java/org/eolang/lints/misc/UnitTestIsNotVerb.java [hunk 1]
```diff
@@ -60,60 +55,52 @@ public final class UnitTestIsNotVerb implements Lint<XML> {
...
-                        ).map(s -> s.toLowerCase(Locale.ROOT))
-                        .toArray(String[]::new)
+            final CoreDocument doc = new CoreDocument(
+                Stream
+                    .concat(
+                        Stream.of("It"),
+                        Arrays.stream(UnitTestIsNotVerb.KEBAB.split(name))
+                    ).map(s -> s.toLowerCase(Locale.ROOT))
+                    .collect(Collectors.joining(" "))
+            );
```
Comment (yegor256) [inline src/main/java/org/eolang/lints/misc/UnitTestIsNotVerb.java:97]: @h1alexbel something is wrong with this formatting. Try this instead:

```
final CoreDocument doc = new CoreDocument(
  Stream
    .concat(
      Stream.of("It"),
      Arrays.stream(UnitTestIsNotVerb.KEBAB.split(name))
    )
    .map(s -> s.toLowerCase(Locale.ROOT))
    .collect(Collectors.joining(" "))
);
```

Make sure dots are vertically aligned. Also, read this: https://www.yegor256.com/2014/10/23/paired-brackets-notation.html

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@h1alexbel looks good, to be honest. One more advantage over `opennlp` is that we don't need to manage model downloading and cache management. It's awesome. However, I would add a `javadoc` comment to the `UnitTestsIsNotVerb.java` to explain why we are using this specific model instead of `opennlp` - you could also include links to the issues that prompted this choice.

Also, there is a significant concern regarding the size of these models. While `opennlp` has a size ~ `4M`, the `stanford-corenlp-4.5.7-models.jar` is `452M` which might pose a problem in some cases. For now It is worth mentioning this in the docs.

**Comment by h1alexbel** [comment]
@volodya-lombrozo updated. Take a look, please

**Review by volodya-lombrozo — APPROVED** [review]
@h1alexbel Looks good to me, thank you for the contribution!

**Comment by h1alexbel** [comment]
@yegor256 take a look, please

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@h1alexbel thanks!

**Pull request merged**